jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -118,6 +119,15 @@ def vue_subset_selected(self, event):
                 f"Failed to extract {event}: {repr(e)}", color='error', sender=self))
 
     def vue_do_aper_phot(self, *args, **kwargs):
+        try:
+            import matplotlib.pyplot as plt
+            from astropy.visualization import quantity_support
+            # stdlib but only import if matplotlib imported
+            import base64
+            from io import BytesIO
+        except ImportError:
+            plt = None

```

Comment:
```suggestion
        except ImportError:
            plt = None
        else:
            # stdlib but only import if matplotlib imported
            import base64
            from io import BytesIO
```
Cleaner to put this in else maybe?

Reply:
Irrelevant now that we switched to bqplot. Thanks anyway!